README.md

```diff
@@ -4,19 +4,15 @@
 
 MRTK is a Microsoft Driven opensource project. 
 
-MRTK-Unity provides a set of foundational components and features to accelerate MR app development in Unity. Latest Release of MRTK (V2) supports HoloLens/HoloLens 2, WMR, and OpenVR platform. Start your project with MRTK, and 
+MRTK-Unity provides a set of foundational components and features to accelerate MR app development in Unity. Latest Release of MRTK (V2) supports HoloLens/HoloLens 2, WMR, and OpenVR platform. 
```

Comment:
I think it would be good to revert these changes and take the changes that Julia just made.

Reply:
And by these I just mean the first two changed sections (feature areas / overview)